### PR TITLE
[conf] allow QtQuick.Layouts 1.1

### PR DESCRIPTION
**Pull request by d0b3rm4n**
[qtquickcontrols] Contributes to JB#24552

### CODE REVIEW

File: allowed_qmlimports.conf [hunk 1]
```diff
@@ -14,6 +14,7 @@ QtQuick.LocalStorage 2.1
 QtQuick.XmlListModel 2.1
 QtQuick.Particles 2.1
 QtQuick.Window 2.1
+QtQuick.Layouts 1.2
```
Comment (tanty) [inline allowed_qmlimports.conf]: I'm not an expert in how this works but, for the current package of qt5-qtquickcontrols-layouts, branched from v5.2.1 tag, the version of the QML module is:

QtQuick.Layouts 1.2

Comment (d0b3rm4n) [inline allowed_qmlimports.conf]: ok will update the PR :)

**Comment by tanty** [comment]
LGTM

**Comment by JussiPakkanen** [comment]
lgtm

**Pull request merged**